c/ldv-consumption/32_7a_cilled_true-unreach-call_linux-3.8-rc1-32_7a-drivers--net--ethernet--intel--e1000e--e1000e.ko-ldv_main1_sequence_infinite_withcheck_stateful.cil.out.c

```diff
@@ -8549,7 +8550,23 @@ struct e1000_info  const  e1000_82583_info  =
 void ldv_check_final_state(void) ;
 void ldv_initialize(void) ;
 extern void ldv_handler_precall(void) ;
+
 extern int __VERIFIER_nondet_int(void) ;
+extern unsigned short int __VERIFIER_nondet_ushort();
+extern _Bool __VERIFIER_nondet_bool();
+extern unsigned long int __VERIFIER_nondet_ulong();
+extern unsigned int __VERIFIER_nondet_uint();
+
+extern void *malloc (size_t __size) __attribute__ ((__nothrow__ , __leaf__)) __attribute__ ((__malloc__)) ;
+void *ldv_successful_malloc(size_t __size) {
+  void *p = malloc(__size);
+  __VERIFIER_assume(p != (void *)0);
+  return p;
+}
+extern void free (void *__ptr) __attribute__ ((__nothrow__ , __leaf__));
+
+int* ts_bus_ctrl_dummy(struct dvb_frontend *p, int i) {};
```

Comment:
you have defined this function but never used

Reply:
Thanks, fixed.